pkg/codegen/dotnet/doc.go

```diff
@@ -50,6 +50,13 @@ func (d DocLanguageHelper) GetDocLinkForResourceType(pkg *schema.Package, _, typ
 	return fmt.Sprintf("/docs/reference/pkg/dotnet/Pulumi%s/%s.html", packageNamespace, typeName)
 }
 
+// GetDocLinkForBuiltInType returns the C# URL for a built-in type.
+// Currently not using the typeName parameter because the returned link takes to a general
+// top -level page containing info for all built in types.
+func (d DocLanguageHelper) GetDocLinkForBuiltInType(typeName string) string {
+	return "https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/built-in-types"
```

Comment:
@sean1588, Why can't we be more specific for C#? For strings, I'd expect to link to https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/reference-types#the-string-type. For bool, I'd expect to link directly to https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/bool. Etc.